### ISSUE TITLE
Training on custom data ???

### ISSUE THREAD

**Issue by dhruvsharma717** [clovaai/CRAFT-pytorch#93]
Hi, I'd like to know if this repo can be used to train CRAFT on custom data using weakly supervised learning ?
I have images with a lot of text on them. All of the text has been tagged but at a sentence level, i.e.,
I have the bounding box information and ground truth label information but at a sentence level.
Can weakly supervised learning be used in this case ?

**Comment by YoungminBaek** [comment]
The training with tagged data at a sentence level is possible using weakly supervised learning. Unfortunately, we do not provide the training code in this repo. Please check out the great re-implementation repo https://github.com/backtime92/CRAFT-Reimplementation.
